backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -200,9 +200,53 @@ public Person addPatient(
       Boolean residentCongregateSetting,
       Boolean employedInHealthcare,
       String preferredLanguage) {
+    return addPatient(
```

Comment:
This diff may be confusing to parse: I've added an additional method signature, which includes Organization. This line is the _original_ method signature, which uses `getCurrentOrganization()` and calls the broader method signature